Source/Umbraco/JS/FormEditor/FormEditorAsync.js

```diff
@@ -275,7 +275,7 @@ addFormEditorCondition("core.fieldisempty", function (rule, fieldValue, formData
 });
 // - "field value is not" condition:
 addFormEditorCondition("core.fieldvalueisnot", function (rule, fieldValue, formData) {
-  return (fieldValue || "").toLowerCase() != (rule.condition.expectedFieldValue || "").toLowerCase();
+  return (fieldValue + "" || "").toLowerCase() != (rule.condition.expectedFieldValue || "").toLowerCase();
```

Comment:
Good point. Shouldn't the right hand side have the same guard, in case the expected field value is a number? That is - `(rule.condition.expectedFieldValue + "" || "")`?